xmake/rules/qt/install/windows.lua

```diff
@@ -0,0 +1,90 @@
+--!A cross-platform build utility based on Lua
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+-- Copyright (C) 2015-2020, TBOOX Open Source Group.
+--
+-- @author      ruki
+-- @file        macosx.lua
+--
+
+-- imports
+import("core.theme.theme")
+import("core.base.option")
+import("core.project.config")
+import("core.project.depend")
+import("detect.sdks.find_vstudio")
+import("private.utils.progress")
+
+-- install application package for windows
+function main(target, opt)
+
+    local installfile = path.join(target:installdir(), "bin", target:basename() .. ".exe")
+
+    -- need re-generate this app?
+    local targetfile = target:targetfile()
+    local dependfile = target:dependfile(targetfile)
+    local dependinfo = option.get("rebuild") and {} or (depend.load(dependfile) or {})
+    if not depend.is_changed(dependinfo, {lastmtime = os.mtime(dependfile)}) then
+        return
+    end
+
+    -- trace progress info
+    -- progress.show(opt.progress, "${color.build.target}generating.qt.app %s.app", target:basename())
```

Comment:
Currently only `after/on_build` has progress information, `after/on_install` does not need it.